corehq/apps/dump_reload/management/commands/dump_domain_data.py

```diff
@@ -49,7 +49,7 @@ def handle(self, domain_name, **options):
 
         for dumper in dumpers:
             filename = _get_dump_stream_filename(dumper.slug, domain_name, utcnow)
-            stream = self.stdout if console else gzip.open(filename, 'wb')
+            stream = self.stdout if console else gzip.open(filename, 'wt')
```

Comment:
This is the operative change https://docs.python.org/3.6/library/gzip.html#gzip.open